### PR TITLE
document the whitelisting engine

### PR DESCRIPTION
**Pull request by harti2006**
Add a README to explain all the feature of the whitelisting engine.

### CODE REVIEW

File: fullstop-whitelist/README.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+ID of an AWS region (see [AWS docs](http://docs.aws.amazon.com/general/latest/gr/rande.html) for a complete list).
+Matches if the violation occurred in this region.
+
+Example: `"eu-central-1"`
+
+### violation_type
+ID of the violation type (use command line tool `fullstop types` to get a complete list).
+Matches if the violation was of exactly this type.
+
+Example: `"UNSECURED_PUBLIC_ENDPOINT"`
```
Comment (sezeryilmaz) [inline fullstop-whitelist/README.md]: We could add a complete json example of a whitelisting rule like this:

{
  "account_id": "123456789",
  "reason": "Aligned with Management. Team is allowed to use other AMI's until the end of the year.",
  "violation_type": "WRONG_AMI",
  "expiry_date": "2017-12-31T00:00:00.000Z"
}

**Comment by sezeryilmaz** [comment]
👍

**Comment by harti2006** [comment]
:+1:

File: fullstop-whitelist/README.md (hunk 1 above)
Comment (harti2006) [reply fullstop-whitelist/README.md]: I added a few complete examples a the bottom of the page

**Comment by ChristianLohmann** [comment]
👍

**Comment by harti2006** [comment]
:+1:

**Pull request merged**